stable/airflow/values.yaml

```diff
@@ -80,7 +80,18 @@ airflow:
   ## Configure pod disruption budget for the scheduler
   podDisruptionBudget:
     maxUnavailable: 1
-
+  ## Add custom connections
+  ## Use this to add Airflow connections for operators you use
+  ## For each connection - the id and type have to be defined.
+  ## All the other parameters are optional
+  ## Connections will be created with a script that is stored
+  ## in a K8s secret and mounted into the scheduler container
+  ## Example:
+  ##   connections:
+  ##   - id: my_aws
+  ##     type: aws
+  ##     extra: '{"aws_access_key_id": "**********", "aws_secret_access_key": "***", "region_name":"eu-central-1"}'
+  connections: {}
```

Comment:
When trying to use this as documented, I get a `Warning: Merging destination map for chart 'airflow'. Cannot overwrite table item 'connections', with non table value: map[]` 
Am I doing this wrong or should this default value be a `[]` ?

Reply:
@ogirardot that is weird. Works fine on my end with curly braces. Are you getting this on the initial install? Can you share your values.yaml?

Reply:
It perhaps worked in earlier versions of helm (if it was less strict), but it's definitely not working on `2.14.1`. We're mixing datatypes here. As per the example above, it expects a list, yet the default value is being assigned as an empty map. As a result of this, it's not possible to pass any connections because of the error @ogirardot mentions.

Reply:
Also, for further clarification, it "works" with empty `{}` braces because the `Range` is empty. But as soon as one attempts to pass a non-empty `[]` list, then it fails.

Reply:
Yes, this should be a list.

Reply:
fixed in 4.0.1